lms/djangoapps/api_manager/courses/views.py

```diff
@@ -1736,7 +1741,8 @@ def get(self, request, course_id):  # pylint: disable=W0613
         if not course_exists(request, request.user, course_id):
             return Response({}, status=status.HTTP_404_NOT_FOUND)
         course_key = get_course_key(course_id)
-        total_possible_completions = float(len(get_course_leaf_nodes(course_key)))
+        course_metadata = CourseAggregatedMetaData.get_from_id(course_key)
+        total_possible_completions = float(course_metadata.total_assessments)
```

Comment:
`get_from_id` method gets a record from CourseAggregatedMetaData if exists otherwise creates one with default values(zeros).  So null check here is unnecessary.